rae_hw/include/rae_hw/peripherals/mic.hpp

```diff
@@ -1,8 +1,15 @@
+// mic_node.hpp
+
 #include <vector>
 #include "rclcpp/rclcpp.hpp"
-#include "audio_msgs/msg/audio.hpp"
```

Comment:
Audio msgs stores pcm data as 8bit - since our mic is 24 bits it is easier to host it as uint32 - so I created a new message which makes that easier. Also helps that we dont need to get audio_msgs from ros gst bridge.